### ISSUE TITLE
styleq with class names?

### ISSUE THREAD

**Issue by nandorojo** [intergalacticspacehighway/react-native-responsive-query#16]
@intergalacticspacehighway now that RNW uses StyleQ with class names under the hood, do you think this library could possibly implement class names instead of data set?

**Comment by intergalacticspacehighway** [comment]
yeah! this [PR](https://github.com/intergalacticspacehighway/react-native-responsive-query/pull/14) uses `className` instead of `dataSet`. Will be merging it soon.
